app/views/containereditor.html

```diff
@@ -56,21 +56,21 @@ <h3 class="available-containers">
           ng-if="refreshingContainers">
         <span class="glyphicon glyphicon-refresh glyphicon-refresh-animate">
         </span>
-        Please wait, retrieving Quay.io and/or Docker Hub containers...
+        Please wait, retrieving Quay.io and/or Docker Hub tools...
```

Comment:
This is the general refresh all workflows message, so I think in this case is makes sense to have the and/or.